src/site/content/en/progressive-web-apps/add-manifest/index.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: post
+title: Add a web app manifest
+authors:
+  - petelepage
+date: 2018-11-05
+updated: 2020-02-14
+description: |
+  The web app manifest is a simple JSON file that tells the browser about your
+  web application and how it should behave when 'installed' on the user's mobile
+  device or desktop.
+---
+
+The web app manifest is a JSON file that tells the browser about your
+Progressive Web App and how it should behave when installed on the user's
+desktop or mobile device. A typical manifest file includes information about
+the app name, icons it should use, and the URL that should be opened when the
+app is launched.
+
+Manifest files are supported in Chrome, Edge, Firefox, UC Browser, Opera,
+the Samsung browser, and Safari partial support.
+
+## Create the manifest.webmanifest file
+
+The manifest file can have any name, but is commonly named
+`manifest.webmanifest` and served from the root (your website's top level
+directory).
+
+```json
+{
+  "short_name": "Weather",
+  "name": "Weather: Do I need an umbrella?",
+  "description": "Weather forecast information",
+  "icons": [
+    {
+      "src": "/images/icons-192.png",
+      "type": "image/png",
+      "sizes": "192x192"
+    },
+    {
+      "src": "/images/icons-512.png",
+      "type": "image/png",
+      "sizes": "512x512"
+    }
+  ],
+  "start_url": "/?source=pwa",
+  "background_color": "#3367D6",
+  "display": "standalone",
+  "scope": "/",
+  "theme_color": "#3367D6"
+}
+```
+
+### Key manifest properties
+
+#### `short_name` and/or `name` {: #name }
+
+You must provide at least the `short_name` or `name` property. If both are
+provided, `short_name` is used on the user's home screen, launcher, or other
+places where space may be limited. `name` is used when the app is installed.
+
+#### `icons` {: #icons }
+
+When a user installs your PWA, you can define a set of icons for the browser
+to use. These icons are used in places like the home screen, app launcher,
```

Comment:
Combine and short the first two sentences as so:

"You can define a set of icons for the browser to use after installation in places like..."

Reply:
Fixed!